osu.Game/Beatmaps/BeatmapSetOnlineStatusExtensions.cs

```diff
@@ -3,6 +3,12 @@
 
 namespace osu.Game.Beatmaps
 {
+    public static class BeatmapSetOnlineStatusExtensions
```

Comment:
Extensions usually go to the bottom of the file rather than top.

Reply:
for what it's worth, we also have [a separate namespace](https://github.com/ppy/osu/tree/master/osu.Game/Extensions) for extension classes.